proto/panacea/oracle/v2alpha2/query.proto

```diff
@@ -23,7 +23,12 @@ service Query {
 
   // OracleRegistration returns a OracleRegistration details.
   rpc OracleRegistration(QueryOracleRegistrationRequest) returns (QueryOracleRegistrationResponse) {
-    option (google.api.http).get = "/panacea/oracle/v2alpha2/oracle_registrations/unique_id/{unique_id}/addresses/{address}";
+    option (google.api.http).get = "/panacea/oracle/v2alpha2/oracle_registrations/{unique_id}/{address}";
+  }
+
+  // OracleRegistrationVote a OracleRegistrationVote details.
+  rpc OracleRegistrationVote(QueryOracleRegistrationVoteRequest) returns (QueryOracleRegistrationVoteResponse) {
+    option (google.api.http).get = "/panacea/oracle/v2alpha2/oracle_registration-votes/{unique_id}/{voting_target_address}/{voter_address}";
```

Comment:
```suggestion
    option (google.api.http).get = "/panacea/oracle/v2alpha2/oracle_registration_votes/{unique_id}/{voting_target_address}/{voter_address}";
```

Reply:
Ah, I missed it.

I just fixed it. (https://github.com/medibloc/panacea-core/pull/396/commits/85f74925848701eb320901a3d09c5375583e6f14)